lib/SIL/IR/OperandOwnership.cpp

```diff
@@ -1033,6 +1033,7 @@ ANY_OWNERSHIP_BUILTIN(IntInstrprofIncrement)
   }
 CONSTANT_OWNERSHIP_BUILTIN(Owned, MustBeInvalidated, COWBufferForReading)
 CONSTANT_OWNERSHIP_BUILTIN(Owned, MustBeInvalidated, UnsafeGuaranteed)
+CONSTANT_OWNERSHIP_BUILTIN(Owned, MustBeInvalidated, GetCurrentAsyncTask)
```

Comment:
This isn't needed. This is only needed to describe the ownership of arguments. No arguments, this isn't needed.

Reply:
Sure, updated with a "should not be called".